README.md

```diff
@@ -35,7 +35,7 @@ Install and try Manticore in a few shell commands (see an [asciinema](https://as
 sudo apt-get update && sudo apt-get install python-pip -y
 
 # Install manticore and its dependencies
-sudo pip2 install manticore
```

Comment:
We want these commands in the quick start to work for people 99.9999% of the time :)

the `--user` command requires ~/.local in your path for things to work, which not everybody has, so that's why in this section we recommend just using sudo, even if it may not be the exact best.